### PR TITLE
feat: opa plugin support timeout

### PR DESCRIPTION
**Pull request by fatelei**

### CODE REVIEW

File: types/plugins/opa/config.proto [hunk 1]
```diff
@@ -17,12 +17,18 @@ syntax = "proto3";
...
 import "validate/validate.proto";
+import "google/protobuf/duration.proto";
 
 option go_package = "mosn.io/htnn/types/plugins/opa";
 
 message Remote {
   string url = 1 [(validate.rules).string = {uri: true}];
   string policy = 2 [(validate.rules).string = {min_len: 1}];
+  google.protobuf.Duration timeout = 3 [(validate.rules).duration = {
+    gte: {
```
Comment (spacewander) [inline types/plugins/opa/config.proto:28]: Would it be better to use `gt: {}` so people can configure a timeout < 1s? For now the default value is already 200ms.

**Comment by spacewander** [comment]
Before I can merge it, could you update doc & fix the lint?

**Comment by fatelei** [comment]
> Before I can merge it, could you update doc & fix the lint?

done

File: site/content/en/docs/reference/plugins/opa.md [hunk 2]
```diff
@@ -25,10 +25,11 @@ Either `remote` or `local` is required.
...
 
-| Name   | Type   | Required | Validation        | Description                                               |
-|--------|--------|----------|-------------------|-----------------------------------------------------------|
-| url    | string | True     | must be valid URI | The url to the OPA service, like `http://127.0.0.1:8181/` |
-| policy | string | True     | min_len: 1        | The name of the OPA policy.                               |
+| Name    | Type   | Required | Validation        | Description                                               |
+|---------|--------|----------|-------------------|-----------------------------------------------------------|
+| url     | string | True     | must be valid URI | The url to the OPA service, like `http://127.0.0.1:8181/` |
+| policy  | string | True     | min_len: 1        | The name of the OPA policy.                               |
+| timeout | string | False    | -                 | http client timeout                                       |
```
Comment (spacewander) [inline site/content/en/docs/reference/plugins/opa.md:32]:
```suggestion
| timeout | [Duration](../../type#duration) | False    |                 | http client timeout                                       |
```

File: site/content/zh-hans/docs/reference/plugins/opa.md [hunk 3]
```diff
@@ -25,10 +25,11 @@ title: OPA
...
 
-| 名称   | 类型   | 必选 | 校验规则          | 说明                                                             |
-|--------|--------|------|-------------------|------------------------------------------------------------------|
-| url    | string | 是   | must be valid URI | 指向 OPA 服务的 url，如 `http://127.0.0.1:8181/`                  |
-| policy | string | 是   | min_len: 1        | OPA 策略的名称                                                   |
+| 名称   | 类型   | 必选 | 校验规则         | 说明                                        |
+|--------|--------|------|------------------|-------------------------------------------|
+| url    | string | 是   | must be valid URI | 指向 OPA 服务的 url，如 `http://127.0.0.1:8181/` |
+| policy | string | 是   | min_len: 1       | OPA 策略的名称                                 |
+| timeout | string | False    | -           | http 客户端超时时间                              |
```
Comment (spacewander) [inline site/content/zh-hans/docs/reference/plugins/opa.md:32]:
```suggestion
| timeout | [Duration](../../type#duration) | 否    |            | http 客户端超时时间                              |
```

File: site/content/en/docs/reference/plugins/opa.md [hunk 4]
```diff
@@ -25,10 +25,11 @@ Either `remote` or `local` is required.
...
 
-| Name   | Type   | Required | Validation        | Description                                               |
-|--------|--------|----------|-------------------|-----------------------------------------------------------|
-| url    | string | True     | must be valid URI | The url to the OPA service, like `http://127.0.0.1:8181/` |
-| policy | string | True     | min_len: 1        | The name of the OPA policy.                               |
+| Name    | Type   | Required | Validation        | Description                                               |
+|---------|--------|----------|-------------------|-----------------------------------------------------------|
+| url     | string | True     | must be valid URI | The url to the OPA service, like `http://127.0.0.1:8181/` |
+| policy  | string | True     | min_len: 1        | The name of the OPA policy.                               |
+| timeout | [Duration](../../type#duration) | False    | -                 | http client timeout                                       |
```
Comment (spacewander) [inline site/content/en/docs/reference/plugins/opa.md:32]:
```suggestion
| timeout | [Duration](../../type#duration) | False    |                  | http client timeout                                       |
```

File: site/content/zh-hans/docs/reference/plugins/opa.md [hunk 5]
```diff
@@ -25,10 +25,11 @@ title: OPA
...
 
-| 名称   | 类型   | 必选 | 校验规则          | 说明                                                             |
-|--------|--------|------|-------------------|------------------------------------------------------------------|
-| url    | string | 是   | must be valid URI | 指向 OPA 服务的 url，如 `http://127.0.0.1:8181/`                  |
-| policy | string | 是   | min_len: 1        | OPA 策略的名称                                                   |
+| 名称   | 类型   | 必选 | 校验规则         | 说明                                        |
+|--------|--------|------|------------------|-------------------------------------------|
+| url    | string | 是   | must be valid URI | 指向 OPA 服务的 url，如 `http://127.0.0.1:8181/` |
+| policy | string | 是   | min_len: 1       | OPA 策略的名称                                 |
+| timeout | [Duration](../../type#duration) | False    | -           | http 客户端超时时间                              |
```
Comment (spacewander) [inline site/content/zh-hans/docs/reference/plugins/opa.md:32]:
```suggestion
| timeout | [Duration](../../type#duration) | 否    |            | http 客户端超时时间                              |
```

**Review by spacewander — APPROVED** [review]

**Comment by spacewander** [comment]
Merged with a little tweak.

**Pull request merged**